sdk/spring/CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ This section includes changes in `spring-cloud-azure-autoconfigure` module.
 #### Bugs Fixed
 - Fixed `GraphClient` exceptions handling when switching `HttpURLConnection` to `RestTemplate`. [#32779](https://github.com/Azure/azure-sdk-for-java/issues/32779)
 
+#### Breaking Changes
+- Deprecated properties for AAD and AAD B2C. [#33538](https://github.com/Azure/azure-sdk-for-java/pull/33538).
```

Comment:
This PR ID should be the current one, please check other changelog items as well.